src/styles/common/alert.scss

```diff
@@ -0,0 +1,16 @@
+.alert-info button{
+  background-color: lightblue;
+  color: white;
+}
+.alert-success button{
+  background-color: lightseagreen;
+  color: white;
+}
+.alert-warning button{
+  background-color: lightsalmon;
+  color: white;
+}
+.alert-danger button{
+  background-color: lightcoral;
+  color: white;
+}
```

Comment:
This is good, I'll take a look later and change the colours to better match our style

Reply:
commit: https://github.com/doubtfire-lms/doubtfire-web/pull/650/commits/62ac3e891f326d92b52d46bbc9b6208a2e96dbec